build_docker_lvl_is.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+set -euo pipefail
+docker build -f docker/Dockerfile.festvox -t harbour.grammatek.com/sim/festvox-build-tools:latest .
+docker build -f docker/Dockerfile.lvl_is --progress=plain -t harbour.grammatek.com/sim/lvl_is:latest .
```

Comment:
Will Grammatek be hosting these images?

Reply:
this is our docker registry, which we have made publicly readable for the sim project. We are hosting these images there so one can just pull them without the need to build these locally